package.json

```diff
@@ -29,7 +29,7 @@
     "semi": false
   },
   "dependencies": {
-    "styled-components": ">=2.0.0"
+    "react-transition-group": "^2.5.0"
```

Comment:
I think it's safe to make `react-transition-group` a dependency of styled-animate.